test/unit/mocks.js

```diff
@@ -52,7 +52,6 @@ define('narrativeMocks', ['jquery', 'uuid', 'narrativeConfig'], ($, UUID, Config
         };
 
         $cellContainer.append($toolbar).append(mockCell.input).append(mockCell.output);
-        $('body').append($cellContainer);
```

Comment:
don't append the cell to the body; let the tests do that explicitly if it is required